string ee_link_name
string finger_link_name
float64[] joint_state_positions
---
sensor_msgs/JointState joint_state
